pkg/minikube/driver/driver.go

```diff
@@ -163,9 +162,9 @@ func FlagDefaults(name string) FlagHints {
 	fh := FlagHints{}
 	if name != None {
 		fh.CacheImages = true
-		// only for kic, till other run-times are available we auto-set containerd.
+		// only for kic, until other runtimes are available we auto-set containerd.
```

Comment:
this comment actually is no long true, we have other run-time,s we can just delete this comment.